docs/pages/guides/localization.mdx

```diff
@@ -0,0 +1,385 @@
+---
+title: Localization
+---
+
+import { ConfigReactNative } from '~/components/plugins/ConfigSection';
+import { SnackInline } from '~/ui/components/Snippet';
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
+
+If you want your app to be easy to use for users that speak different languages or come from different cultures, you should localize it.
+
+Localizing an app makes it adapt to the locale of the user's device. The app will show translations and currencies that the user knows and understands. Numbers, lists and more will format in a way that the user is used to.
+
+## Getting the user's language
+Use the [`expo-localization`](../versions/latest/sdk/localization) module to get the user's current language.
+
+<Terminal cmd={['$ npx expo install expo-localization']} />
+
+```ts
+import { getLocales } from "expo-localization";
+
+const deviceLanguage = getLocales()[0].languageCode;
+```
+
+Expo localization returns the current locale based on the system settings of the device.
+
+On new iOS and Android versions, app language can be set per app, so you usually don't need to build a custom UI to allow users to change the current locale inside of your app. 
+
+Sometimes, it makes sense to build UI to allow the user to set other localization preferences on a per-app basis. As a general rule you should allow the user to change:
+ - Localized units if your app makes at least a moderate use of them (such as metric/imperial measurements, currency, temperature and more)
+ - Other preferences if there's no API to get the default value on platforms you want to support (check [`expo-localization`](../versions/latest/sdk/localization) API documentation for details)
+
+## Translating an app
+
+Creating and managing translations quickly becomes a large task. 
+
+You can handle translations manually, but it's best to use a library to handle this for you.
+
+Let's make our app support English and Japanese. To achieve this install the i18n package `i18n-js`:
```

Comment:
```suggestion
Creating and managing translations quickly becomes a large task. You can handle translations manually, but it's best to use a library to handle this for you.

Let's make the app support English and Japanese. To achieve this install the i18n package `i18n-js`:
```

The introduction to this section looked scattered. I think we can join the first two sentences to make them resemble the subject of this paragraph (the problem it solves).